packages/seatmaps-client/package.json

```diff
@@ -29,6 +35,7 @@
   "scripts": {
     "dev": "vite",
     "build": "pnpm build:cjs && pnpm build:esm",
+    "vite-build": "vite build",
```

Comment:
Don't forget to include your new build into the build command so it'll get run in CI.